### ISSUE TITLE
Update archives hierarchy

### ISSUE THREAD

**Issue by maximearmstrong** [MobilityData/mobility-database-catalogs#57]
**What problem is your feature request trying to solve?**
 As we migrate from the Wikibase instance, we may need to update the Mobility Archives hierarchy/file system to reflect our current needs.
 
The current bucket and file system for a source is

Storage level. For each source:
- Bucket: {source_archives_id}_archives_{some_download_date}
    - File: {sha1_hash}.{extension}
- Bucket: {source_archives_id}_archives_{another_download_date}
    - File: {sha1_hash}.{extension}
- Bucket: {source_archives_id}_latest
    - File: {sha1_hash}.{extension}

The system was built this way because we wanted to give each object a different [life cycle] (https://cloud.google.com/storage/docs/lifecycle) and we thought it should be done per bucket and not per object. After re-reading the documentation, I believe that the lifecycle rules apply to a bucket, but that each file in the bucket is assigned a lifecycle class based on how often it is used. 

So we should restructure this, and also remove the dependency on the `archives_id` that are linked to the Wikibase instance.

**Describe the solution you'd like**

Storage level: 
- Bucket: archives
    - Object folder (one per source): {source_filename_as_used_in_the_catalogs}/{download_date}/{sha1_hash}.{extension}
        - Object: {download_date}/{sha1_hash}.{extension}
        OR
        - Object: {sha1_hash}.{extension}
- Bucket: latest
    - Object (one per source): {source_filename_as_used_in_the_catalogs}.{extension} 
- Bucket: in-use?
    - Object (one per source): {source_filename_as_used_in_the_catalogs}.{extension} 

**Describe alternatives you've considered**

**How will we know when this is done?**
A clear and concise user story or list of user stories that must be met for the issue to be completed. A user story follows the structure: “As a [persona], I [want to], [so that].”

**Additional context**

**Comment by emmambd** [comment]
@maximearmstrong Is this still to be done?

**Comment by maximearmstrong** [comment]
I think this is something we will do if we have historical datasets and the in-use dataset for a source. Is it still in our roadmap?

**Comment by emmambd** [comment]
@maximearmstrong Yes. I just wasn't sure if this was still applicable, but it's good to know this is still needed to capture historical datasets.
